src/js/common/constants.js

```diff
@@ -0,0 +1,20 @@
+export const MESSAGE = {
+    OPERAND_FIRST: '숫자를 먼저 입력한 후 연산자를 입력해주세요!',
+    MAX_LENGTH: '숫자는 세 자리까지만 입력 가능합니다!',
+    ONLY_TWO_OPERAND: '2개의 숫자 조합만 계산 가능합니다!',
+};
+
+export const OPERATIONS = {
+    PLUS: '+',
+    MINUS: '-',
+    DIVIDE: '/',
+    MULTIPLE: 'X',
+    CALCULATE: '=',
+};
+
+export const CALCULATE = {
+    [OPERATIONS.PLUS]: (a, b) => +a + +b,
+    [OPERATIONS.MINUS]: (a, b) => +a - +b,
+    [OPERATIONS.MULTIPLE]: (a, b) => +a * +b,
+    [OPERATIONS.DIVIDE]: (a, b) => Math.floor(+a / +b),
+};
```

Comment:
```suggestion
export const CALCULATE = {
    [OPERATIONS.PLUS]: (a, b) => Number(a) + Number(b),
    [OPERATIONS.MINUS]: (a, b) => Number(a) - Number(b),
    [OPERATIONS.MULTIPLE]: (a, b) => Number(a) * Number(b),
    [OPERATIONS.DIVIDE]: (a, b) => Math.floor(Number(a) / Number(b),
};
```
이렇게 명시적으로 Number라고 표현해주면 어떨까요?

그리고 Math.floor의 경우 음수에 대해서는 생각하는 것과 다른 동작이 수행될 수 있습니다.
<img width="138" alt="image" src="https://user-images.githubusercontent.com/18749057/158188428-ecd754ec-d4f8-48cc-a6c2-375e12db9747.png">

이럴 때 `~~` 연산을 하면 아예 소숫점을 버린답니다!

참고로 `~`는 1의 보수 연산을 수행하고, 이 과정에서 모든 소숫점을 버립니다.
`~~` 처럼 1의 보수 연산을 두 번 수행하게 되면 소숫점만 버리는 연산을 수행할 수 있답니다!

<img width="129" alt="image" src="https://user-images.githubusercontent.com/18749057/158189038-8883cba1-8aac-4571-879d-cac4174255bb.png">